hydrolib/core/io/ini/models.py

```diff
@@ -285,6 +286,53 @@ def validate(cls, v):
                 )
         return super().validate(v)
 
+    @staticmethod
+    def _get_friction_root_validator(
+        frictionid_attr: str,
+        frictiontype_attr: str,
+        frictionvalue_attr: str,
+    ):
+        """
+        Make a root_validator that verifies whether the crosssection definition (subclass)
+        has a valid friction specification.
+        Supposed to be embedded in subclasses for their friction fields.
+
+        Args:
+            frictionid_attr: name of the frictionid attribute in the subclass.
+            frictiontype_attr: name of the frictiontype attribute in the subclass.
+            frictionvalue_attr: name of the frictionvalue attribute in the subclass.
+
+        Returns:
+            root_validator: to be embedded in the subclass that needs it.
+        """
+
+        def validate_friction_specification(cls, values):
+            """
+            The actual validator function.
+
+            Args:
+            cls: The subclass for which the root_validator is called.
+            values (dict): Dictionary of values to create a CrossSectionDefinition subclass.
+            """
+            frictionid = values.get(frictionid_attr) or ""
+            frictiontype = values.get(frictiontype_attr) or ""
+            frictionvalue = values.get(frictionvalue_attr) or ""
```

Comment:
You _could_ also remove the ` or ""` and check for None, but it does not really matter :)

Reply:
With this code I wanted to catch if the user somehow specifies the empty string for frictionids:
```
frictionid = 
```